lib/gateway.js

```diff
@@ -0,0 +1,21 @@
+var util = require('util');
+var Device = require('./device');
+var Peripheral = require('./peripheral');
+var debug = require('debug')('losant:gateway');
+
+function Gateway(options) {
+  Device.call(this, options);
+
+  this.peripherals = [];
+}
+
+util.inherits(Gateway, Device);
+
+Gateway.prototype.addPeripheral = function(deviceId) {
+  debug('Adding peripheral to gateway: ' + deviceId);
+  var peripheral = new Peripheral({ id: deviceId, mqtt: this.mqtt });
+  this.peripherals.push(peripheral);
```

Comment:
should peripherals be a map (key being id) so if you call addPeripheral multiple times on the same id you get the same instance?

also, should a peripheral have a reference back to its gateway instance?